index.html

```diff
@@ -270,87 +329,289 @@
+[=inappropriate=] ways, with no way to discover the discrimination or seek relief
+([[?Democratic-Data]]).
+
+What we consider is therefore not just the relation between the [=people=] who share data
+and the [=parties=] that invite that disclosure ([[?Relational-Turn]]), but also between the [=people=]
+who may find themselves categorised indirectly even without sharing data. One key understanding here
+is that such relations may persists even when data is [=de-identified=]. What's more, such
+categorisation of people, voluntary or not, will change the way in which the world operates.
+This can produce self-reinforcing loops that can have deleterious effects both individual and
+collective ([[?Seeing-Like-A-State]]).
+
+In general, collective issues in [=data=] require collective solutions. The goal of
+[=data governance=] in Web standards is the development of structural controls
+in [=user agents=] and the creation of institutions that can handle issues of [=privacy=].
+[=Governance=] will often struggle to achieve its goals if it works primarily by
+increasing <em>individual</em> control. A collective approach reduces the cost of control.
+
+Collecting data at large scales can have significant pro-social outcomes. Problems tend to
+emerge when entities take part in dual-use collection in which [=data=] is [=processed=]
+for collective benefit but also for [=self-dealing=] [=purposes=].
+The [=self-dealing=] [=purposes=] will be justified as bankrolling the pro-social outcomes,
+which, without collective oversight, cannot be legitimate.
+
+### Transparency and Research {#transparency}
+
+While transparency is rarely of any help in informing the individual choices that [=people=] may
+make or in increasing their [=autonomy=], it is instrumental to support the accountability work
+that can be performed by researchers and reporters as it plays a critical role in informing
+collective decision-making with respect to privacy [=rules=]. This consideration extends the
+TAG's resolution on a [Strong and Secure Web Platform](https://www.w3.org/blog/2015/11/strong-web-platform-statement/)
+to ensure that <q>broad testing and audit continues to be possible</q> where
+[=information flows=] and automated decisions are involved.
+
+Such transparency can only function if there are strong rights of access to data (including data
+derived from one's personal data) as well as mechanisms to explain the outcomes of automated
+decisions.
+
+## People's Agents {#user-agents}
+
+The <dfn>user agent</dfn> acts as an intermediary between a [=person=] (its [=user=]) and the web.
+[=User agents=] implement, to the extent possible, the [=rules=] that collective governance
+establishes in favour of individuals. They seek to prevent the creation of asymmetries of
+information, and serve their [=user=] by providing them with automation, to rectity
+[=automation asymmetries=]. Where possible, they protect their [=user=] from receiving
+[=inappropriate=] information.
```

Comment:
Although "inappropriate" links to a definition that makes it reasonable here, I think the plain-English meaning has us endorsing anti-obscenity censorship. Maybe

```suggestion
[=automation asymmetries=]. Where possible, they protect their [=user=] from receiving
intrusive messages.
```